src/packages/core/property-action/shared/property-action-menu/property-action-menu.element.ts

```diff
@@ -10,24 +10,25 @@ import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
 @customElement('umb-property-action-menu')
 export class UmbPropertyActionMenuElement extends UmbLitElement {
 	#actionsInitializer?: UmbExtensionsElementInitializer<ManifestTypes, 'propertyAction'>;
+	#value: unknown;
+	#propertyEditorUiAlias = '';
 
 	@property({ attribute: false })
 	public set value(value: unknown) {
-		this._value = value;
+		this.#value = value;
 		if (this.#actionsInitializer) {
 			this.#actionsInitializer.properties = { value };
 		}
 	}
 	public get value(): unknown {
-		return this._value;
+		return this.#value;
 	}
 
-	private _value?: unknown;
-
-	#propertyEditorUiAlias = '';
-
 	@property()
 	set propertyEditorUiAlias(alias: string) {
+		// If there is an existing initializer, we need to dispose it.
+		this.#actionsInitializer?.destroy();
```

Comment:
Actually this should already be happening, since the initialiser always gets the same alias. See line 43